src/rawchannel.cc

```diff
@@ -78,6 +79,7 @@ class RawChannel : public Nan::ObjectWrap
 {
 private:
   static Nan::Persistent<v8::Function> constructor;
+  int Flag_CANFD_Used=0;         // Add a global Flag to treat the information according the interface capability, see Init function
```

Comment:
Please move member variables to the designated section (after private: around line 454).

Also, members follow a specific naming conventions (camel case), in your case it should be call m_IsCanFdUsed

Reply:
done